package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "@gnosis.pm/zodiac",
-  "version": "3.0.0",
+  "version": "4.0.0",
```

I think this should only be a minor bump, since users will be able to upgrade their @gnosis.pm/zodiac dependency without changing anything in their contracts.

```suggestion
  "version": "3.2.0",
```